Userland/Applications/HexEditor/HexEditorWidget.cpp

```diff
@@ -372,3 +372,16 @@ void HexEditorWidget::set_search_results_visible(bool visible)
     m_layout_search_results_action->set_checked(visible);
     m_search_results_container->set_visible(visible);
 }
+
+void HexEditorWidget::drop_event(GUI::DropEvent& event)
+{
+    event.accept();
```

Comment:
Right now this also brings the hex editor to the front when the dropped data is incompatible, e.g. when dropping cells from `Spreadsheet`. Maybe we should only accept the event and activate the app after the check?

Reply:
Good idea! I’ve basically copy-pasted the code from the Sound Player so I’ll fix it there as well. :^)